package.json

```diff
@@ -13,14 +13,21 @@
   },
   "homepage": "https://github.com/FintelmannLabDevelopmentTeam/ostrich#readme",
   "dependencies": {
-    "chai": "^4.3.4"
+    "chai": "^4.3.4",
+    "mkdirp": "^1.0.4"
```

Comment:
This is still only a dev-dependency, right?

Reply:
Yes, I will change it.